usr/src/cmd/fs.d/smbclnt/fksmbcl/.dbxrc

```diff
@@ -0,0 +1,24 @@
+
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2017 Nexenta Systems, Inc.  All rights reserved.
+#
+
+set -o emacs
+
+# fksmbcl is always 32-bit
```

Comment:
Given that we only build 64-bit kernels these days, is there a reason that our "faker" is not 64-bit as well?

Reply:
Yeah, because it links in a bunch of user-level libraries that aren't ported to 64-bit (and where doing that is "hard" or just not really worth the trouble to do).